src/main/java/brightspark/sparkshammers/integration/crafttweaker/CraftTweakerIntegration.java

```diff
@@ -0,0 +1,12 @@
+package brightspark.sparkshammers.integration.crafttweaker;
+
+import crafttweaker.CraftTweakerAPI;
+import net.minecraftforge.fml.common.Optional;
+
+public class CraftTweakerIntegration {

```

Comment:
You don't need to manually register the class, you can simply add
[`@ZenRegister`](http://crafttweaker.readthedocs.io/en/latest/#Dev_Area/ZenAnnotations/Annotation_ZenRegister/) to your main class

That way, you don't have to check if the mod is loaded either